inference-engine/tests/functional/shared_test_classes/src/base/layer_test_utils.cpp

```diff
@@ -60,7 +61,8 @@ void LayerTestsCommon::Run() {
 
     try {
         LoadNetwork();
-        for (size_t i = 0; i < targetStaticShapes.size(); i++) {
+        size_t i = 0;
+        do {
             index = i;
```

Comment:
Agree, having 'index' makes unclear what this cycle does (and why result will be changed on each iteration)

Reply:
`index` is using to generate correct blobs and reshape `ngraph::function`. CPU plugin required changes contains one dynamic shape and some number of static shape. It's reusable variable. 
Discussed offline @ilya-lavrenov

Reply:
we can fix all places and make them to accept `i` instead of global field `index`

Reply:
In such case I should update signature in all places in OpenVINO and KMB. It means that PR merge will take much more time.